website/docs/cloud-docs/vcs/github-app.mdx

```diff
@@ -60,6 +60,10 @@ HCP Terraform requests GitHub authorization from each user, displaying a pop-up
 
 ![Screenshot: GitHub asking whether you want to authorize "Terraform Cloud by HashiCorp".](/img/docs/gh-app-authorize.png)
 
+After initial configuration of GitHub App is performed, and therefore VCS provider instance created, GitHub App option will become grayed out and unavailable. You can modify existing configuration without removing it from GitHub, and re-creating.
+To add new repositories, use GitHub repository Settings tab(must be repository owner for it to be visible and accessible) -> Github Apps -> Terraform Cloud -> Configure -> Repository access
+In order to pull up the pop-up window again, use HCP Terraform -> Workspaces -> New -> Version Control Workflow -> GitHub App -> click GitHub Organization right above the repository list on the left -> Add another organization. 
+
 Once you authorize the app, you can use GitHub in any of your HCP Terraform organizations without needing to re-authorize.
 
```

Comment:
Adding this all in one suggestion for easier readability. I suggest adding bold for UI elements, rewording for active voice, and adding a link to your existing "Create a workspace" article so we don't have to repeat the instructions of how to create a workspace with a VCS connection:
```suggestion
After installing the GitHub App and creating your VCS provider instance, you cannot reinstall the application again. However, you can modify your existing GitHub App configuration.

If you are a repository owner, you can adjust an existing Github App configuration by:
1. Click a repository's **Settings** tab.
1. Select **GitHub Apps** in the sidebar.
1. Next to **Terraform Cloud**, click **Configure**. 
1. Underneath **Repository access**, adjust the repositories your GitHub app can access.

Now that your Github app has access to your desired repository, you can [create a new workspace](/terraform/cloud-docs/workspaces/create#create-a-workspace) with your existing, newly updated GitHub App connection.
```